leftwm/src/config/mod.rs

```diff
@@ -469,11 +469,17 @@ impl leftwm_core::Config for Config {
     }
 
     fn border_width(&self) -> i32 {
-        self.theme_setting.border_width
+        self.theme_setting.border_width.unwrap_or(1)
     }
 
     fn margin(&self) -> Margins {
-        match self.theme_setting.margin.clone().try_into() {
+        match self
+            .theme_setting
+            .margin
+            .clone()
```

Comment:
Do we stell bed this clone? Might be with the `unwarp_or` we could loose this.

Reply:
If I remove the .clone() it does not compile. `unwrap_or(...)` moves the object.

Reply:
Yeah, was just wondering why `border_width` has no clone.

Btw, can't we use `unwrap_or_default`?

Reply:
It's an enum, so I would have to write a manual implementation. Also all other values also use `.unwrap_or()`, example in line 531 with false.

Reply:
I think this is to clarify the defaults